app/assets/javascripts/discourse/app/components/reviewable-item.js

```diff
@@ -12,6 +12,12 @@ import showModal from "discourse/lib/show-modal";
 
 let _components = {};
 
+const pluginReviewableParams = [];
+
+export function addPluginReviewableParam(param) {
```

Comment:
This API should be specific to the type too.

Reply:
@eviltrout updated. New api looks like:

`api.addPluginReviewableParam("ReviewableCategoryExpertSuggestion", "group_id");`